libvcx/src/credential_def.rs

```diff
@@ -169,7 +208,21 @@ impl CredentialDef {
 
 fn _parse_revocation_details(revocation_details: &str) -> VcxResult<RevocationDetails> {
     serde_json::from_str::<RevocationDetails>(&revocation_details)
-        .to_vcx(VcxErrorKind::InvalidRevocationDetails, "Cannot deserialize RevocationDeltas")
+        .to_vcx(VcxErrorKind::InvalidRevocationDetails, "Cannot deserialize RevocationDetails")
+}
+
+fn _maybe_set_url(rev_reg_def_json: &str, revocation_details: &RevocationDetails, rev_reg_id: &str) -> VcxResult<String> {
+    // TODO: Deserialization into RevRegDef loses ver field
```

Comment:
If there's a `ver` field, why don't we put it on `RevocationRegistryDefinition` so we could use it?

Reply:
It would be especially nicer if we could return `RevocationRegistryDefinition ` instead of String